### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by AlecAivazis**
This PR makes a few updates to the project including adding a prettier config, as well as fixing issue #101.

### CODE REVIEW

File: src/util/createReducer.test.js [hunk 1]
```diff
@@ -234,4 +234,16 @@ describe('createReducer', function() {
         // make sure the computed lessThan object matches exepctation
         expect(getIs(currentType, breakpoints)).toEqual(expected)
     })
+
+    it('is object can refer to non-numerical values', function() {
+        // the expectation lessThan
```
Comment (deadNightTiger) [inline src/util/createReducer.test.js]: Copy-pasted comment? Shouldn't it be `is`?

File: src/util/createReducer.test.js [hunk 2]
```diff
@@ -234,4 +234,16 @@ describe('createReducer', function() {
...
+        const expected = {
+            small: false,
+            medium: false,
+            large: false,
+            foo: true
+        }
+        // make sure the computed lessThan object matches exepctation
```
Comment (deadNightTiger) [inline src/util/createReducer.test.js]: Copy-pasted again?

File: src/util/createReducer.js [hunk 3]
```diff
@@ -98,14 +98,8 @@ export function getLessThan(currentMediaType, breakpointOrder) {
...
     return transform(breakpoints, (result, breakpoint, mediaType) => {
-        // if the breakpoint is a number
-        if (typeof breakpoint === 'number' && breakpoints[mediaType]) {
-            // store wether or not it is less than the breakpoint
-            result[mediaType] = mediaType === currentMediaType
-            // handle non numerical breakpoints specially
-        } else {
-            result[mediaType] = false
-        }
+        // store wether or not the current media type matches the breakpoint in the mapping
```
Comment (deadNightTiger) [inline src/util/createReducer.js]: `wether` - typo?

**Comment by AlecAivazis** [comment]
Thanks for the review @deadNightTiger! I fixed the typos you found

**Comment by AlecAivazis** [comment]
Since you done so much work helping me with this library I have invited you @deadNightTiger to be an official collaborator. Hopefully you don't mind. I'd love a second pair of eyes as I continue to maintain this if you are open to it!

**Pull request merged**